components/kyma-environment-broker/internal/process/provisioning/setup_backup.go

```diff
@@ -0,0 +1,95 @@
+package provisioning
+
+import (
+	"fmt"
+
+	"github.com/kyma-incubator/compass/components/kyma-environment-broker/internal/ptr"
+
+	"time"
+
+	"github.com/kyma-incubator/compass/components/kyma-environment-broker/internal"
+	"github.com/kyma-incubator/compass/components/kyma-environment-broker/internal/hyperscaler"
+	"github.com/kyma-incubator/compass/components/kyma-environment-broker/internal/process"
+	"github.com/kyma-incubator/compass/components/kyma-environment-broker/internal/storage"
+	"github.com/kyma-incubator/compass/components/provisioner/pkg/gqlschema"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	// the time after which the operation is marked as expired
+	SetupUpBackupTimeOut = 1 * time.Hour
+)
+
+type SetupBackupStep struct {
+	bucketName       string
+	zone             string
+	operationManager *process.OperationManager
+	accountProvider  hyperscaler.AccountProvider
+}
+
+func (s *SetupBackupStep) Name() string {
+	return "Setup_Backup"
+}
+
+func NewSetupBackupStep(os storage.Operations, accountProvider hyperscaler.AccountProvider) *SetupBackupStep {
+	return &SetupBackupStep{
+		operationManager: process.NewOperationManager(os),
+		accountProvider:  accountProvider,
+	}
+}
+
+func (s *SetupBackupStep) SetupBackupStep() {
+
+}
+
+func (s *SetupBackupStep) Run(operation internal.ProvisioningOperation, log logrus.FieldLogger) (internal.ProvisioningOperation, time.Duration, error) {
+	if time.Since(operation.UpdatedAt) > SetupUpBackupTimeOut {
```

Comment:
use `RetryOperation` from `operation.go`. Let's have a common pattern for handling timeouts 👍

Reply:
I think I can remove this for now since here I am only setting the overrides. I would need retries in the next issue: #901